10-hibernate-spring-data/src/main/java/ru/sber/controllers/CartController.java

```diff
@@ -0,0 +1,68 @@
+package ru.sber.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import ru.sber.entities.Cart;
+import ru.sber.entities.Payment;
+import ru.sber.entities.Product;
+import ru.sber.services.CartService;
+import ru.sber.services.CartServiceImpl;
+import ru.sber.services.ProductService;
+
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequestMapping("carts")
+public class CartController {
+    private final CartServiceImpl cartServiceImpl;
+
+    @Autowired
+    public CartController(CartServiceImpl cartServiceImpl) {
+        this.cartServiceImpl = cartServiceImpl;
+    }
+
+    @PostMapping("/{cartId}/products/{productId}")
+    public void addProductToCart(@PathVariable long cartId, @PathVariable long productId) {
+        log.info("Товар с id {} добавлен в корзину с id {}", productId, cartId);
```

Comment:
Ты тут пишешь, что товар добавлен, но по факту он будет добавлен после этого сообщения)
А вдруг что-то пойдет не так и он не добавится?
Лучше напиши 
```suggestion
        log.info("Товар с id {} добавляется в корзину с id {}", productId, cartId);
```

Reply:
И в других местах тоже самое

Reply:
Исправил